redfish-core/lib/log_services.hpp

```diff
@@ -2838,7 +2838,7 @@ inline void requestRoutesDBusEventLogEntry(App& app)
             if constexpr (BMCWEB_IBM_MANAGEMENT_CONSOLE)
             {
                 asyncResp->res.jsonValue["Oem"]["OpenBMC"]["@odata.type"] =
```

Comment:
The problem here is this is still under OpenBMC

Reply:
I'll rename `Oem/OpenBMC/ `to` Oem/IBM/ `for IBMLogEntry schema

Reply:
I'll keep `Oem/OpenBMC/`  and move IBMLogEntry.json to `oem/openbmc/json-schema/OpenBMCLogEntry.json`